docs/client-sdk/javascript/tutorials/build-key-xmtp-chat-features-in-a-lens-app.md

```diff
@@ -236,6 +244,8 @@ for await (const conversation of client.conversations.stream()) {
 }
 ```
 
+Messages sent within a conversation are not returned by this method. Use the `conversations.streamAllMessages` to get all messages in the new Lens conversation.
```

Comment:
Also made this update based on Nick's reply - but it probably needs some tweaking to make it work in the context of this tutorial? I'm not sure - so just flagging for help! <3

Reply:
In this context I think we want to get users to use `conversation.streamMessages()` which will just stream the messages for the one new conversation.